samples/snippets/pom.xml

```diff
@@ -40,6 +40,12 @@
       <artifactId>protobuf-java-util</artifactId>
       <version>4.0.0-rc-2</version>
     </dependency>
+    <dependency>
+        <groupId>org.glassfish</groupId>
+        <artifactId>javax.json</artifactId>
+        <version>1.1</version>
+      <type>jar</type>
```

Comment:
Can we use either Jackson2 or GSON here?

Reply:
Done.